charts/hawkbit-update-server/values.yaml

```diff
@@ -46,11 +51,13 @@ env:
   springRabbitmqHost: "hawkbit-update-server-rabbitmq"
   springRabbitmqUsername: "hawkbit"
   springRabbitmqPassword: "hawkbit"
-  securityUserName: "admin"
+  springsecurityUserName: "admin"
```

Comment:
Has to be `springSecurityUserName` (as referenced in README and `deployment.yaml`)